hack/release.toml

```
@@ -31,6 +31,12 @@ Talos is built with Go 1.21.6.
         title = "Device Selectors"
         description = """\
 Talos Linux now supports `physical: true` qualifier for device selectors, it selects non-virtual network interfaces (i.e. `en0` is selected, while `bond0` is not).
+"""
+
+    [notes.dns-resolve-cache]
+        title = "DNS Caching"
+        description = """\
+Talos now caches DNS lookups for host and host-networked pods. This brings us in line with the behavior of other Kubernetes distributions.
```

```suggestion
Talos Linux now  provides a caching DNS resolver for host workloads (including host networking pods). It can be disabled with:

```yaml
machine:
   features:
       blah: blah
```
```